test/extended/util/framework.go

```diff
@@ -320,6 +367,18 @@ func WaitForSamplesImagestream(ctx context.Context, oc *CLI, imagestream string,
 	return pollErr
 }
 
+// ImportImageStream imports the provided image reference to an imagestream with
+// the given name in the oc client's current namespace.
+func ImportImageStreams(oc *CLI, imageStreamName, imageRef string) error {
+	err := oc.Run("import-image").Args(
+		imageStreamName,
+		fmt.Sprintf("--from=%s", imageRef),
+		"--import-mode=PreserveOriginal",
+		"--confirm").Execute()
```

Comment:
From prior feedback I received - invoking the `oc` binary in this manner is discouraged for running OpenShift tests moving forward. When possible (as is the case here), the preference is to use Kubernetes clients to create/update the respective k8s objects.

Example (pseduo-code)

```go
is, err := oc.ImageClient().ImageV1().Imagestreams(namespace).Create(ctx, isObj, metav1.CreateOptions{})
```

For ImageStreams, you will want to use the openshift Image api clients to create an ImageStream object. See https://github.com/openshift/api/blob/master/image/v1/types.go#L233 for more info.